Remove Red Hat branding and added downstream docs

ritz303 opened the pull request:
## Description
Remove Red Hat branding from documentation and ported over the downstream content.

Fixes: Jira [DBAAS-1232](https://issues.redhat.com/browse/DBAAS-1232)

## Type of change
- [ ] Content edits to the YAML QSG files.
- [ ] Ported over the downstream Quick Start Guide and Reference Guide.

## Checklist (Definition of Done)
<!-- Please strikethrough options not relevant using two tildes ~~Text~~. Do not delete non relevant options -->
- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

mrrajan commented on controllers/reconcilers/quickstartinstallation/connecting-an-application-to-a-database-instance-using-the-topology-view-quick-start.yaml:
```diff
@@ -10,12 +10,12 @@ metadata:
     data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAuMCAwLjAgMTQ1LjQ0NjE5NDIyNTcyMTggMTQ0LjM3Nzk1Mjc1NTkwNTUiIGZpbGw9Im5vbmUiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLWxpbmVjYXA9InNxdWFyZSIgc3Ryb2tlLW1pdGVybGltaXQ9IjEwIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48Y2xpcFBhdGggaWQ9InAuMCI+PHBhdGggZD0ibTAgMGwxNDUuNDQ2MiAwbDAgMTQ0LjM3N ...
   description: 'Use the Topology view to connect database instances to your applications.'
-  prerequisites: ["Access to the OpenShift console.", "Installation of the Red Hat OpenShift Database Access add-on.", "A cloud-hosted database provider added to your project."]
+  prerequisites: ["Access to the OpenShift console.", "Installation of the OpenShift Database Access add-on.", "A cloud-hosted database provider added to your project."]
```
Please update the "add-on" to "operator"

ritz303 replied:
@mrrajan : Good catch! Thanks for taking a look.

tchughesiv reviewed (changes requested):
we may have to move away from using mongo as the example, depending on what they decide wrt merging our code upstream. maybe the safer route is to use RDS since we're more likely to package that with our release?

tchughesiv reviewed (changes requested):
also, could you please remove any references you find around this being a "service preview" release... or beta release? thanks

ritz303 commented on controllers/reconcilers/quickstartinstallation/installing-the-openshift-database-access-operator-quick-start.yaml:
```diff
@@ -0,0 +1,37 @@
+  icon: >-
+    data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAuMCAwLjAgMTQ1LjQ0NjE5NDIyNTcyMTggMTQ0LjM3Nzk1Mjc1NTkwNTUiIGZpbGw9Im5vbmUiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLWxpbmVjYXA9InNxdWFyZSIgc3Ryb2tlLW1pdGVybGltaXQ9IjEwIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48Y2xpcFBhdGggaWQ9InAuMCI+PHBhdGggZD0ibTAgMGwxNDUuNDQ2MiAwbDAgMTQ0LjM3N ...
+  description: 'Install the OpenShift Database Access operator in your cluster.'
+  prerequisites: ["OpenShift Container Platform 4.10 or higher.", "Running a OpenShift Dedicated, or an OpenShift Service on Amazon Web Services (AWS).", "AWS account and credentials."]
```
@tchughesiv : Thanks for the feedback!  I've updated the prereqs accordingly.

tchughesiv commented on controllers/reconcilers/quickstartinstallation/installing-the-openshift-database-access-operator-quick-start.yaml:
```diff
@@ -15,7 +15,7 @@ spec:
     data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAuMCAwLjAgMTQ1LjQ0NjE5NDIyNTcyMTggMTQ0LjM3Nzk1Mjc1NTkwNTUiIGZpbGw9Im5vbmUiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLWxpbmVjYXA9InNxdWFyZSIgc3Ryb2tlLW1pdGVybGltaXQ9IjEwIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48Y2xpcFBhdGggaWQ9InAuMCI+PHBhdGggZD0ibTAgMGwxNDUuNDQ2MiAwbDAgMTQ0LjM3N ...
   description: 'Install the OpenShift Database Access operator in your cluster.'
-  prerequisites: ["OpenShift Container Platform 4.10 or higher.", "Running a OpenShift Dedicated, or an OpenShift Service on Amazon Web Services (AWS).", "AWS account and credentials."]
+  prerequisites: ["Either OpenShift Container Platform or Origin Key Distribution (OKD) 4.10 or higher"]
```
Is that what OKD stands for? I always thought it was OpenShift Community Distribution

tchughesiv reviewed (changes requested):
@ritz303 i think we'll need to remove mentions of mongodb atlas from the docs. let's use RDS as an example instead and in docs/references

also, please remove mentions of this release being "service preview". tks!

ritz303 commented:
@tchughesiv :  I thought I got all of the "Service Preview" instances, but I did find one more, so thank you for double checking me.  I've also replaced all mentions of MongoDB and replaced them with Amazon RDS.  Please let me know if I've missed anything.

ritz303 commented:
@tchughesiv : Thanks for taking a look!  I've removed the draft status.  I'm curious, can you do a squash on my commits before merging this PR into main?

ritz303 reviewed (commented):
Finished reviewing the documentation.
